Public/Src/FrontEnd/TypeScript.Net/TypeScript.Net/TypeChecking/Checker.cs

```diff
@@ -15925,13 +15925,23 @@ private IType CheckSwitchExpression(ISwitchExpression node, ITypeMapper contextu
                     TypeToString(expressionType));
             }
 
-            if (node.Clauses.Count == 0)
+            if (!node.Clauses.Any(clause => !clause.IsDefaultFallthrough))

```

Comment:
>!node.Clauses.Any(clause => !clause.IsDefaultFallthrough)) [](start = 16, length = 58)

reword in terms of .All to simplify negations?

Reply:
Open to suggestion for the expression. 
It is checking that there must exists at least one clause that is not a default... Not sure how to do with all...

---
In reply to: [274186828](https://github.com/Microsoft/BuildXL/pull/123#discussion_r274186828) [](ancestors = 274186828)